### PR TITLE
better cni compatibility 

### PR DESCRIPTION
**Pull request by YaoZengzeng**

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
#192

**Comment by YaoZengzeng** [comment]
#191

File: pkg/hyper/network.go [hunk 1]
```diff
@@ -81,56 +86,368 @@ func toAPIProtocol(protocol Protocol) v1.Protocol {
...
 		return err
 	}
 
 	return nil
 }
+
+func buildNetworkInfo(bridgeName string, cniResult *current.Result) *NetworkInfo {
+	ret := &NetworkInfo{}
+
+	ret.BridgeName = bridgeName
```
Comment (feiskyer) [inline pkg/hyper/network.go]: Should also generate the Ifname with same bridge suffix, e.g. for bridge br1a089c6f, generate a tap1a089c6f.

**Pull request merged**